src/test/java/de/zalando/ep/zalenium/servlet/LiveNodeServletTest.java

```diff
@@ -1,131 +1,98 @@
-
-        request = mock(HttpServletRequest.class);
-        response = mock(HttpServletResponse.class);
-
-        when(request.getParameter("refresh")).thenReturn("1");
-        when(request.getServerName()).thenReturn("localhost");
-        when(response.getOutputStream()).thenReturn(TestUtils.getMockedServletOutputStream());
-    }
-
-    @Test
-    public void addedNodesAreRenderedInServlet() throws IOException {
-
-        LivePreviewServlet livePreviewServletServlet = new LivePreviewServlet(registry);
-
-        livePreviewServletServlet.doPost(request, response);
-
-        String responseContent = response.getOutputStream().toString();
-        assertThat(responseContent, containsString("Zalenium Live Preview"));
-        assertThat(responseContent, containsString("http://localhost:40000"));
-        assertThat(responseContent, containsString("http://localhost:40001"));
-        assertThat(responseContent, containsString("/vnc/host/localhost/port/50000/?nginx=localhost:50000&view_only=true'"));
-        assertThat(responseContent, containsString("/vnc/host/localhost/port/50000/?nginx=localhost:50000&view_only=false'"));
-        assertThat(responseContent, containsString("/vnc/host/localhost/port/50001/?nginx=localhost:50001&view_only=true'"));
-        assertThat(responseContent, containsString("/vnc/host/localhost/port/50001/?nginx=localhost:50001&view_only=false'"));
-    }
-
-    @Test
-    public void postAndGetReturnSameContent() throws IOException {
-
-        LivePreviewServlet livePreviewServletServlet = new LivePreviewServlet(registry);
-
-        livePreviewServletServlet.doPost(request, response);
-        String postResponseContent = response.getOutputStream().toString();
-
-        livePreviewServletServlet.doGet(request, response);
-        String getResponseContent = response.getOutputStream().toString();
-        assertThat(getResponseContent, containsString(postResponseContent));
-    }
-
-    @Test
-    public void noRefreshInHtmlWhenParameterIsInvalid() throws IOException {
-        when(request.getParameter("refresh")).thenReturn("XYZ");
-
-        LivePreviewServlet livePreviewServletServlet = new LivePreviewServlet(registry);
-
-        livePreviewServletServlet.doPost(request, response);
-        String postResponseContent = response.getOutputStream().toString();
-        assertThat(postResponseContent, containsString("<meta http-equiv='refresh' content='XYZ' />"));
-    }
-    
-    @After
-    public void tearDown() throws MalformedObjectNameException {
-        ObjectName objectName = new ObjectName("org.seleniumhq.grid:type=RemoteProxy,node=\"http://localhost:40000\"");
-        new JMXHelper().unregister(objectName);
-        objectName = new ObjectName("org.seleniumhq.grid:type=RemoteProxy,node=\"http://localhost:40001\"");
-        new JMXHelper().unregister(objectName);
-        ContainerFactory.setContainerClientGenerator(originalContainerClient);
-    }
+//    private GridRegistry registry;
```

Comment:
@dspasojevic What's the plan for this test?

Reply:
I will see whether I can rebirth this test.

Reply:
I *think* that this test was written before the registry implementation could be swapped out. I'll see if I can rewrite to use a simpler grid impl so the docker container client doesn't have to be mocked.

Does that sound sensible to you @diemol ?

Reply:
Re-enabled.